benchmark_runner/main/main.py

```diff
@@ -169,9 +169,9 @@ def upgrade_ocp_bare_metal(step: str):
     if step == 'run_bare_metal_ocp_upgrade':
         if not bare_metal_operations.is_ocp_already_upgraded(oc):
             bare_metal_operations.run_ocp_upgrade(oc)
-            # The LSO/ODF upgrade must be run manually after the OCP upgrade for the channel version; it won't upgrade automatically.
-            bare_metal_operations.install_ocp_resources(resources=['lso'], upgrade_version=lso_version)
-            bare_metal_operations.install_ocp_resources(resources=['odf'], upgrade_version=odf_version)
+            # Upgrade manually to 4.18 didnt work
+            #bare_metal_operations.install_ocp_resources(resources=['lso'], upgrade_version=lso_version)
+            #bare_metal_operations.install_ocp_resources(resources=['odf'], upgrade_version=odf_version)
```

Comment:
Can this be conditioned on the upgrade version?  If somebody else wants to use this with, say, 4.17 or 4.19 there's no reason that this should be disabled for non-broken versions.